CONTRIBUTING.md

```diff
@@ -29,6 +29,69 @@ Documentation for the in-the-wild deployments lives in the
 
 See [DEVELOP.md](./DEVELOP.md) for more information on how to run the development environment.
 
+## Proposing Changes
+The Wormhole project understands that contributions come in all shapes and sizes and from both from casual contributions (eg. fix a typo) to advanced contributions (eg. add a new chain integration).  Understanding this, the Wormhole project aims to offer the following paths to make contributions to Wormhole.
+### GitHub Pull Request (aka: Casual Contribution)
+**Objective:** Allow any casual contribution to the project using the default tooling provided by GitHub.
+
+**Examples of casual contribution**
+- Adjustments to documentation
+- Adjustments to configuration parameters
+- Fixing a script or otherwise non-mission critical piece of code
```

Comment:
I would also suggest adding self-contained single commit changes to this list.  If a change is not self-contained or requires multiple commits then people should be using the gerrit flow.